math/binomialcoefficient.go

```diff
@@ -0,0 +1,32 @@
+// binomialcoefficient.go
+// description: Returns C(n, k)
+// details:
+// a binomial coefficient C(n,k) gives number ways
+// in which k objects can be chosen from n objects.
+// wikipedia: https://en.wikipedia.org/wiki/Binomial_coefficient
+// author: Akshay Dubey (https://github.com/itsAkshayDubey)
+// see binomialcoefficient_test.go
+
+package math
+
+import (
+	"errors"
+)
+
+// C is Binomial Coefficient function
+// This function returns C(n, k) for given n and k
+func C(n int, k int) (int, error) {
+	var ErrPosArgsOnly error = errors.New("arguments must be positive")
```

Comment:
This needs to be outside the function not local to the function so that you can use it inside the test rather than creating a new error inside test.